arrows/ocv/tests/test_crop_chips.cxx

```diff
@@ -0,0 +1,182 @@
+/*ckwg +29
+ * Copyright 2013-2016 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
+ *    to endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * \file
+ * \brief test OCV image class
+ */
+
+#include <vital/plugin_loader/plugin_manager.h>
+
+#include <vital/types/image_container.h>
+#include <vital/types/image_container_set.h>
+
+#include <arrows/ocv/crop_chips.h>
+#include <arrows/ocv/image_container.h>
+#include <arrows/ocv/image_io.h>
+
+#include <gtest/gtest.h>
+
+
+int
+main(int argc, char* argv[])
+{
+  ::testing::InitGoogleTest( &argc, argv );
+  return RUN_ALL_TESTS();
+}
+
+using namespace kwiver::vital;
+
+TEST(crop_chips, factory)
+{
+  using namespace kwiver::arrows;
+
+  kwiver::vital::plugin_manager::instance().load_all_plugins();
+
+  algo::crop_chips_sptr algo = kwiver::vital::algo::crop_chips::create("ocv");
+  EXPECT_NE(nullptr, algo) << "Unable to create crop_chips algorithm of type ocv";
+  algo::crop_chips* algo_ptr = algo.get();
+  ASSERT_TRUE(typeid(*algo_ptr) == typeid(ocv::crop_chips)) << "Factory method did not construct the correct type";
+}
+
+
+namespace {
+
+// TODO: The following functions create dummy image data for implementing
```

Comment:
Feel free to make a `test_image.h` in this same directory for this purpose...

The non-OpenCV-specific flavors are already in `arrows/tests/test_image.h`.